api/db/populate-messages.js

```diff
@@ -16,19 +16,21 @@ const createMessage = async (messageParams) => {
 
   const user =
     message.friendship.receivant.toString() === message.user.toString()
-      ? message.friendship.receivant
-      : message.friendship.requestant;
-  await createNotification(
-    {
-      user,
-      type: notificationTypes[8],
-      read: false,
-      referencedUser: message.user,
-      referencedFriendship: message.friendship,
-    },
-    displayLogs
-  );
-  notificationsCount++;
+      ? message.friendship.requestant
+      : message.friendship.receivant;
+  if (!message.friendship.accepted) {
```

Comment:
This check shouldn't go here but somewhere below -- what doesn't have to be created if the friendship request hasn't been accepted is the actual message instead of its notification